recipes/scicookie/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scicookie" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scicookie-{{ version }}.tar.gz
+  sha256: b1f6c94b9370a8ef21e5591ff091dc931a0dec806bca7245a5741739c190a214
+
+build:
+  skip: true  # [py>=400 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
note sure if we should use pip or poetry here 
if it doesn't work with pip .. maybe you can try with poetry install